app/views/reading_collections/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= @collection.name %>
+<% @collection.articles.each do |article| %>
+  <div class="article">
+    <%= article.title %>
```

Comment:
Awesome. I know Joel will have to make changes, but this setup is perfect for what I did with _new.